components/authz-service/README.md

```diff
@@ -615,6 +615,18 @@ For others, it will be necessary to add and remove policies
 to lock the system down so that known users are authorized to access
 specific resources.
 
+#### Adding New Default Policies
+
+To add a new default policy, the following is needed:
+
+1. A new IAM v1 policy: these are defined in [migrations](storage/postgres/migration/sql/),
+   and the [v1 constants package](constants/v1/constants.go#L3-L4).
+2. A new IAM v2 policy: they are defined in [server/v2/system](server/v2/system.go#L31).
+3. Migration logic: When upgrading from IAM v1 to v2 (v2.1), v1 policies are
+   converted. The conversion logic needs to be made aware of the new v1 policies, and
+   how (and if) they are to be migrated (if they hadn't been deleted). The procedure for
```

Comment:
I think we want "if they haven't been deleted" . . . but that is confusing, also. You mean if they haven't been migrated?

Reply:
You're right about the grammar, for course -- what this was trying to describe is that the migration code only migrates v1 policies that haven't been deleted. There used to be shipped-with-A2 v1 policies that the user could delete. Not all "default policies" were non-deletable.